src/ui/digital-exchange/CategoryTabsContainer.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import { convertToQueryString, FILTER_OPERATORS } from '@entando/utils';
+
+import { getDECategoryList } from 'state/digital-exchange/categories/selectors';
+import { fetchDEComponents, setDEFilters } from 'state/digital-exchange/components/actions';
+import { fetchDECategories } from 'state/digital-exchange/categories/actions';
+import CategoryTabs from 'ui/digital-exchange/CategoryTabs';
+
+const FIELD_OPERATORS = {
+  category: FILTER_OPERATORS.LIKE,
+};
+
+export const mapDispatchToProps = dispatch => ({
+  onWillMount: () => (dispatch(fetchDECategories())),
+  onSelect: (category) => {
+    const filters = {
+      formValues: { category: category ? [category] : [] },
+      operators: FIELD_OPERATORS,
+    };
+    // TODO proposal for https://jira.entando.org/browse/EN-2325: pass filters to fetchDEComponents and dispatch set filters action there
```

Comment:
lets avoid TODO comments in code unless we know for a fact that they are gonna be gone within a few PRs